### PR TITLE
Removed start term from participants pages

### PR DESCRIPTION
**Pull request by lazydevorg**
### Context

- Ticket: CST-807

### Changes proposed in this pull request

- Removed references to start term
- Removed links about edit start term

### CODE REVIEW

File: app/components/schools/participants/status_table_row.rb [hunk 1]
```diff
@@ -26,9 +26,7 @@ def participant_is_on_a_cip?
         profile.school_cohort.school_chose_cip?
       end
 
-      def date_column_value
```
Comment (xswoodsx) [inline app/components/schools/participants/status_table_row.rb:29]: Can this just be removed and the have an empty value in the <td> where it currently is?

File: app/views/schools/participants/show.html.erb [hunk 2]
```diff
@@ -43,33 +43,6 @@
...
-      <div class="govuk-summary-list__row">
-        <dt class="govuk-summary-list__key">
-          Induction start
-        </dt>
-        <dd class="govuk-summary-list__value">
-          <%= @first_induction_record&.start_date.to_date.to_s(:govuk) %>
-        </dd>
-        <dd class="govuk-summary-list__actions">
-        </dd>
-      </div>
```
Comment (tonyheadford) [inline app/views/schools/participants/show.html.erb:72]: Should this be removed?

Comment (lazydevorg) [reply app/views/schools/participants/show.html.erb:72]: Mili confirmed it needs to be removed

File: app/controllers/schools/participants_controller.rb [hunk 3]
```diff
@@ -160,14 +136,4 @@ def participant_mentor_form_params
...
   end
-
-  def start_term_form_params
-    params.require(:participant_start_term_form).permit(:start_term)
-  rescue ActionController::ParameterMissing
-    nil
-  end
-
-  def build_start_term_form
-    ParticipantStartTermForm.new(start_term_form_params)
```
Comment (xswoodsx) [inline app/controllers/schools/participants_controller.rb:171]: Is this used anywhere else?  If not we can delete the file for the form to? 
https://github.com/DFE-Digital/early-careers-framework/blob/develop/app/forms/participant_start_term_form.rb

File: app/views/schools/participants/show.html.erb (hunk 2 above)
Comment (tonyheadford) [reply app/views/schools/participants/show.html.erb:72]: Really? The induction start date bit too?

Comment (lazydevorg) [reply app/views/schools/participants/show.html.erb:72]: it was a misunderstanding. The induction start should not be removed. I'll revert the change.
Thanks @tonyheadford 👍

**Review by xswoodsx — APPROVED** [review]
Looks good, just need to re-run the tests. My PR had this before and just needed a rerun

**Pull request merged**